src/main/java/duke/duke/Duke.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+import duke.info.exception.DukeException;
+import duke.info.task.Calendar;
+
+import duke.commands.Command;
+
+import duke.parser.Parser;
+
+import duke.storage.Storage;
+
+import duke.ui.Ui;
+
+public class Duke {
+    private Storage storage;
+    private Calendar calendar;
+    private Ui ui;
+
+    /**
+     * Runs the Duke program using the run() method specified.
+     * @param args - command line arguments
```

Comment:
Should there be no spacing between the description and parameter section? I noticed the issue in other places also.
```suggestion
     * Runs the Duke program using the run() method specified.
     *
     * @param args - command line arguments
```